code/modules/antagonists/renegade/renegade.dm

```diff
@@ -0,0 +1,102 @@
+/datum/antagonist/renegade
+	name = "Renegade"
+	show_in_antagpanel = TRUE
+	antagpanel_category = "Other"
+	job_rank = ROLE_RENEGADE
+	antag_hud_name = "renegade"
+	show_name_in_check_antagonists = TRUE
+	roundend_category = "renegade"
+	count_against_dynamic_roll_chance = FALSE
+	silent = TRUE //not actually silent, because greet will be called by the trauma anyway.
+	suicide_cry = "FOR MY SAFETY!!"
+	preview_outfit = /datum/outfit/renegade
+	var/datum/brain_trauma/special/renegade/trauma
+
+/datum/antagonist/renegade/admin_add(datum/mind/new_owner,mob/admin)
+	var/mob/living/carbon/C = new_owner.current
+	if(!istype(C))
+		to_chat(admin, "[roundend_category] come from a brain trauma, so they need to at least be a carbon!")
+		return
+	if(!C.get_organ_by_type(/obj/item/organ/internal/brain)) // If only I had a brain
+		to_chat(admin, "[roundend_category] come from a brain trauma, so they need to HAVE A BRAIN.")
+		return
+	message_admins("[key_name_admin(admin)] made [key_name_admin(new_owner)] into [name].")
+	log_admin("[key_name(admin)] made [key_name(new_owner)] into [name].")
+	//PRESTO FUCKIN MAJESTO
+	C.gain_trauma(/datum/brain_trauma/special/renegade)//ZAP
+
+/datum/antagonist/renegade/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/renegade.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
+	owner.announce_objectives()
+	to_chat(owner, span_boldannounce("As a Renegade you're a secondary antagonist! You are allowed to escalate when there is any threat to your life, protecting it by any means! But you're not a murderer, you're just extremely scared for your life and paranoid of your surroundings."))
```

Comment:
This needs to use the config policy system.

Reply:
> This needs to use the config policy system.

How you use it?

Reply:
seems like it's done by default on all antags, so you can just have the headmins add this config when the pr is merged.

Reply:
(you need to remove this line)